simulation/simulation_interface/proto/simulation_api_schema.proto

```diff
@@ -98,7 +113,10 @@ message UpdateFrameResponse {
  **/
 message SpawnVehicleEntityRequest {
   traffic_simulator_msgs.VehicleParameters parameters = 1; // Parameters of vehicle entity.
-  bool is_ego = 2;                                    // If true, the entity is Ego vehicle. (Autoware vehicle.)
+  bool is_ego = 2;                                         // If true, the entity is Ego vehicle. (Autoware vehicle.)
+  string asset_key = 3;                                    // Asset key of the entity simulator entity
+  geometry_msgs.Pose pose = 4;                             // Entity initial pose
```

Comment:
As agreed in [ZMQ ss2 <-> AWSIM/simple_simulator document](https://docs.google.com/document/d/1lQIVPp-eHlSMdf8ocGOOIWrZQRSbBYOB6BIkStUJEbo/edit) the field is supposed to be `pose` not `initial_pose`. I would like to double-check if this change is requested. If yes, I will change it.

Reply:
If we have to indicate in the comment that the data member is "Entity initial pose", we thought it would be better to use the variable name initial_pose from the beginning.
However, if the interface specification has already been agreed upon, it is not so much that I want to change it now (even if it takes time).

Therefore, I withdraw this change request.